go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/hashicorp/go-memdb v1.3.4
 	github.com/ipfs/go-datastore v0.6.0
 	github.com/ipfs/go-ds-badger2 v0.1.3
+	github.com/libp2p/go-libp2p-core v0.20.1
```

Comment:
You're importing `go-libp2p-core` somewhere, which is deprecated.  Find that import and change it to `go-libp2p/core/<...>`